### PR TITLE
[11.0][MIG] mail restrict follower selection

### PR DESCRIPTION
**Pull request by phucngta**

### CODE REVIEW

File: mail_restrict_follower_selection/__init__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+##############################################################################
```
Comment (jcdrubay) [inline mail_restrict_follower_selection/__init__.py]: Should we switch to the shorter version?

```
# 2015 Therp BV <http://therp.nl>
# 2017 Komit <http://www.komit-consulting.com>
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).`
```

Also, I am not sure if it is ok to add Komit  to the copyrights for a migration.

@OCA/core-maintainers should there be any mention about the good practice in the migration guideline page: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0 ?

File: mail_restrict_follower_selection/README.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+* Holger Brunn <hbrunn@therp.nl>
+* Nguyen Tan Phuc <phuc.nt@komit-consulting.com>
+
+Funders
+-------
+
+The development of this module has been financially supported by:
+
+* Komit https://komit-consulting.com
```
Comment (jcdrubay) [inline mail_restrict_follower_selection/README.rst]: I think that Funders section can be removed as it is not applicable.

Komit is not a funder as their was no financial contribution, only time (and sweat).

File: mail_restrict_follower_selection/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Restrict follower selection",
+    "version": "11.0.1.0.0",
+    "author": "Komit, "
```
Comment (jcdrubay) [inline mail_restrict_follower_selection/__manifest__.py]: I don't think that Komit is not the author as Komit only takes care of the migration.

For a migration, I think that Komit should only be added as a contributor, in the README.rst

File: mail_restrict_follower_selection/models/mail_wizard_invite.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+##############################################################################
+from lxml import etree
+from odoo import models, api
```
Comment (jcdrubay) [inline mail_restrict_follower_selection/models/mail_wizard_invite.py]: Please order the imports:

`from odoo import api, models`

File: mail_restrict_follower_selection/__init__.py (hunk 1 above)
Comment (phucngta) [reply mail_restrict_follower_selection/__init__.py]: In pull request https://github.com/OCA/web/pull/738, @moylop260 asked me to keep original Copyright so i don't want to switch this to the shorter version.

Comment (hbrunn) [reply mail_restrict_follower_selection/__init__.py]: it's fine to shorten this, `keeping the copyright` means not to delete the names. So I agree with @jcdrubay's proposal

File: mail_restrict_follower_selection/__manifest__.py (hunk 3 above)
Comment (hbrunn) [reply mail_restrict_follower_selection/__manifest__.py]: yes, author stays Therp BV and the OCA

**Review by hbrunn — APPROVED** [review]

**Review by njeudy — APPROVED** [review]
Code and test on my local database

**Comment by hbrunn** [comment]
please fix

```
module mail_restrict_follower_selection: creating or updating database tables
<string>:3: (WARNING/2) Explicit markup ends without a blank line; unexpected unindent.
2018-03-06 17:57:58,873 166 WARNING openerp_test odoo.addons.base.module.module: docutils' system message present: <system_message level="2" line="3" source="<string>" type="WARNING"><paragraph>Explicit markup ends without a blank line; unexpected unindent.</paragraph></system_message>
```

then we can merge

**Comment by phucngta** [comment]
@hbrunn It's fixed

**Pull request merged**